widget/embedded/src/store/quote.ts

```diff
@@ -224,8 +224,8 @@ export const useQuoteStore = createSelectors(
         set((state) => ({
           inputAmount: amount,
           ...(!amount && {
-            outputAmount: new BigNumber(0),
-            outputUsdValue: new BigNumber(0),
+            outputAmount: null,
+            outputUsdValue: null,
```

Comment:
![Screenshot from 2024-07-28 10-10-40](https://github.com/user-attachments/assets/b63f09e2-cb8a-487d-8c40-d834854f7d71)
![Screenshot from 2024-07-28 10-11-03](https://github.com/user-attachments/assets/c214d422-7d12-4684-a696-d6733ffda814)

By changing this part, this bug occurs, The output USD value is erased once the input value has been cleared. (Previously, this bug was fixed by altering this section.)

Reply:
Tnx, Fixed.